core/controllers/editor.py

```diff
@@ -380,8 +380,8 @@ def get(self, exploration_id):
             version = exploration.version
 
         # If the title of the exploration has changed, we use the new title.
-        filename = 'oppia-%s-v%s.zip' % (
-            utils.to_ascii(exploration.title.replace(' ', '')), version)
+        filename = 'oppia-%s-v%s.zip' % (exploration.title.replace(
```

Comment:
Have you tested the case where the title contains unicode characters?

Reply:
Yep tested it with this string: u'¡Hola!'

Reply:
One question ... could you change the value used in tests to this string instead? Just for robustness against the future.

Reply:
I think the to_ascii was required as it escaped the unicode characters. Without it, the outputs seems different. For example the filename gets converted to \xc2\xa1Hola! when it should be \xa1Hola!. If I wrap the to_ascii method around the whole title, the output would be `attachment; filename=b'oppia-Hola!-v1.zip'`. Is it OK then?

Reply:
I think your last suggestion seems reasonable to me.

Reply:
Done